client/src/components/atoms/Tag.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { css } from "@emotion/react";
+import { theme } from "@/theme";
+import type { Color } from "@/theme";
+
+const colors = (color: Color, bg: Color) =>
+  css({
+    color: theme.colors[color],
+    backgroundColor: theme.colors[bg],
+  });
+
+type TagTypes = "public" | "private" | "identified" | "anonymous" | "votable";
+
+const tagNames = {
+  public: "공개",
+  private: "비공개",
+  identified: "기명",
+  anonymous: "무기명",
+  votable: "투표 가능",
+} as const;
```

Comment:
`as const` 키워드를 사용하는것보단 '`TagTypes`를 키로 가지고 value가 string인 object'를 의미하는 `Record<TagTypes, string>`을 쓰면 좋을 것 같아요!

이렇게 하면 `TagTypes`가 바뀌었을 때 대응하기 용이해집니당
```suggestion
const tagNames: Record<TagTypes, string> = {
  public: "공개",
  private: "비공개",
  identified: "기명",
  anonymous: "무기명",
  votable: "투표 가능",
};
```
전 리뷰에도 슥끼어들어가있긴 했는데 설명이 부족했던 것 같네요ㅠㅠ